src/hiccup/compiler.clj

```diff
@@ -201,15 +212,15 @@
   [[tag attrs & content :as element]]
   (cond
     (every? literal? element)
-      ::all-literal                    ; e.g. [:span "foo"]
+    ::all-literal                    ; e.g. [:span "foo"]
```

Comment:
Try to avoid changing the indentation of rows unaffected by the change, even if the indentation is wrong.